### PR TITLE
DEV 1306 fix signature image resize issue

### PR DESCRIPTION
**Pull request by ahsinsiddique**

### CODE REVIEW

File: src/app/mail/mail-settings/addresses-signature/addresses-signature.component.ts [hunk 1]
```diff
@@ -13,6 +13,50 @@ import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
 import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
 import { Subject } from 'rxjs/internal/Subject';
 
+import Quill from 'quill';
+import ImageResize from 'quill-image-resize-module';
+
+
+// Register quill modules and fonts and image parameters
+Quill.register('modules/imageResize', ImageResize);
```
Comment (techevanglist) [inline src/app/mail/mail-settings/addresses-signature/addresses-signature.component.ts:21]: move the shared code into shared service  and only reqister modules in this component
Only follow code needs to be in this component I think, the rest can be used in different places in app.
```
Quill.register('modules/imageResize', ImageResize);
Quill.register(ImageFormat, true);
```

Comment (ahsinsiddique) [reply src/app/mail/mail-settings/addresses-signature/addresses-signature.component.ts:21]: yes, Okay so now I'll move it to shared service

File: src/app/store/services/shared.service.ts [hunk 2]
```diff
@@ -17,6 +17,48 @@ import { PlanType, PricingPlan } from '../datatypes';
 import { MailContactComponent } from '../../mail/mail-contact/mail-contact.component';
 import { SaveContactComponent } from '../../mail/mail-contact/save-contact/save-contact.component';
 
+import Quill from 'quill';
+import ImageResize from 'quill-image-resize-module';
+
+
+// Register quill modules and fonts and image parameters
+Quill.register('modules/imageResize', ImageResize);
```
Comment (techevanglist) [inline src/app/store/services/shared.service.ts:25]: you shouldn't register it here, both register events should be done in component, this should be moved back.
```
Quill.register('modules/imageResize', ImageResize);
```

Comment (ahsinsiddique) [reply src/app/store/services/shared.service.ts:25]: Okay

**Pull request merged**